testing/vcs/test_script_for_py.py

```diff
@@ -0,0 +1,15 @@
+import vcs
+
+list=["boxfill","meshfill","isofill","textcombined","texttable","vector"]
+for obj in list:
+    s=''
+    tc=''
+    if obj == 'textcombined':
+        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left')
+        tc="'EXAMPLE_tt','EXAMPLE_tto'"
+    code="s=vcs.get%s(%s)" % (obj, tc)
+    exec(code)
+    try:
+        s.script(obj+"_script.py")
+    except:
+        raise("Python script creation broke on "+obj+".script()")
```

Comment:
This doesn't work, for a couple of reasons:

- raise is a keyword, not a function. This is just a misleadingly worded version of `raise "Python script creation broke on..."`.
- You can only raise objects that derive from the `BaseException` class in python. This usually means something like: `raise ValueError("Python script creation broke on...").

Moreover, you should just `print "Python script creation broke on..."` and do `sys.exit(1)`.